pkg/operator2/starter.go

```diff
@@ -186,10 +187,11 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	)
 
 	// TODO remove this controller once we support Removed
-	managementStateController := management.NewOperatorManagementStateController("authentication", operatorClient, ctx.EventRecorder)
+	managementStateController := management.NewOperatorManagementStateController("authentication", operatorClient, controllerCtx.EventRecorder)
 	management.SetOperatorNotRemovable()
 	// TODO move to config observers
 	// configobserver.NewConfigObserver(...)
+	ctx := controllerCtx.Ctx
```

Comment:
why not just pass it directly?

Reply:
I only wanted to make it more readable. `ctx.Ctx` does not seem to look good for reading. Since it is used in many places, created a new variable instead of passing directly.